components/typography/stories/template.js

```diff
@@ -106,7 +112,413 @@ export const Template = (args = {}, context = {}) => {
+			direction: "column",
+			heading: "Light sans serif",
+			content: RegularBoldItalicGroup({...args, semantics: "detail", weight: "light"}, context)
+		})}
+		${Container({
+			direction: "column",
+			heading: "Light Serif",
+			content: RegularBoldItalicGroup({...args, semantics: "detail", glyph: "serif", weight: "light"}, context)
+		})}
+	`
+});
+
+export const DocsCodeVariants = (args, context) => Container({
+	withBorder: false,
+	content: html`
+		${Container({
+			direction: "column",
+			heading: "Default/sans serif",
+			content: RegularBoldItalicGroup({...args, semantics: "code"}, context)
+		})}
+	`
+});
+
+export const DocsHeadingBodyPairing = (args, context) => Container({
+	direction: "column",
+	withBorder: false,
+	content: html`
+	${Template({
+		...args,
+		context,
+		content: [
+			{
+				semantics: "heading",
+				content: "Lorem ipsum dolor",
+				customStyles: {"margin-block-start": "0"},
+			},
+			{
+				semantics: "body",
+				content: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec eleifend est mollis ligula lobortis, tempus ultricies sapien lacinia. Nulla ut turpis velit. Sed finibus dapibus diam et sollicitudin. Phasellus in ipsum nec ante elementum congue eget in leo. Morbi eleifend justo non rutrum venenatis. Fusce cursus et lectus eu facilisis. Ut laoreet felis in magna dignissim feugiat. Ut et lectus finibus, aliquet mauris eu, tincidunt mi. Donec scelerisque orci sit amet venenatis luctus. Morbi eget lacus est. Duis iaculis magna quis aliquam lacinia."
+			}
+		]
+	})}`
+});
+
+// ********* Template groups for Internationalization ********* //
+
+const InternationalizedRegularBoldItalicGroup = (args, context) => Container({
+	direction: "column",
+	withBorder: false,
+	wrapperStyles: {"row-gap": "0"},
+	content: html`
+		${Template({
+			...args,
+			context,
+			content: [
+				{
+					content: html`${args.internationalizedContent.regular} (regular ${args.semantics})`,
+					customStyles: {"margin-block-start": "0"},
+				},
+				{
+					content: html`${args.internationalizedContent.emphasized} (emphasized ${args.semantics})`,
+					variant: ["emphasized"],
+				},
+				{
+					content: html`${args.internationalizedContent.strong} (strong ${args.semantics})`,
+					variant: ["strong"],
+				}
+			]
+		})}`
+});
+
+// @todo these are carryover from the docs site; need to ensure accuracy in translation
+// they might all mean "heading"
+const JapaneseInternationalizedContent = {
+	regular: "見出し",
+	emphasized: "見出し 重点",
+	strong: "見出し 強い強調",
+};
+
+const ArabicInternationalizedContent = {
+	regular: "القسم",
+	emphasized: "القسم تشديد",
+	strong: "القسم تأكيد قو",
+};
+
+const HebrewInternationalizedContent = {
+	regular: "גוף הדגשות",
+	emphasized: "גוף חזק",
+	strong: "גוף חזק",
+};
+
+export const DocsInternationalizedHeadingVariants = (args, context) => Container({
+	withBorder: false,
+	content: html`
+		${Container({
+			direction: "column",
+			heading: "Japanese (Default)",
```

Comment:
```suggestion
			heading: "Japanese (default)",
```

I believe this format would be sentence case per the style guide. Would you mind double checking the headings?

Reply:
Great catch, thanks!